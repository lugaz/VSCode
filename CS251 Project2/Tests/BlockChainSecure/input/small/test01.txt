21 empty
addBlock rS2PU84J8E 45.290835 478
addBlock MyXLzQ 85.044792 158
updateBlock MyXLzQ 22.786342
getBlock MyXLzQ
addBlock hvs1T 105.959138 467
getEarliestBlock
getEarliestBlock
getBlock eotBx
addBlock Yu -51.662591 477
getEarliestBlock
updateEarliestBlock -172.503869
removeEarliestBlock
