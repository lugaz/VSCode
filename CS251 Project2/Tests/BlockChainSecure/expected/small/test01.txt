23
true 3
true 12
MyXLzQ 22.786342 479
true 10
hvs1T 105.959138 467
hvs1T 105.959138 467
null
true 0
hvs1T 105.959138 467
Yu -51.662591 477
